libs/traits/Cargo.toml

```diff
@@ -43,6 +44,7 @@ std = [
   "sp-std/std",
   "cfg-primitives/std",
   "scale-info/std",
+  "serde/std",
```

Comment:
Since it's optional, shouldn't we apply `?`
```suggestion
  "serde?/std",
```

Reply:
Wondering why adding this broke tests?

Reply:
I think:
- `serde?/std` adds `serde/std` in case `serde` is enable
- `serde/srd` enables `serde` and adds `serde/std`